fastlane/Fastfile

```diff
@@ -109,7 +123,7 @@ platform :ios do
     next unless verify(title: title, tag_prefix: tag_prefix)
     increment_version()
     build_universal_framework()
-    sh('zip -r ~/Desktop/TwilioVerify.framework.zip ~/Desktop/TwilioVerify.framework')
```

Comment:
Do we plan to have both (framework and xcframework) to support Carthage?

Reply:
Just `xcframework`, even if Carthage does not support the raw xcframework yet, it is capable to build it from source. Also the Carthage team is already working on supporting the raw xcframeworks, so is no ideal to support old fat framework anymore.